doc/source/news.rst

```diff
@@ -3,6 +3,15 @@
 News
 ====
 
+.. _release-12-09:
+
+Release 12.09 - 2022-10-28
+--------------------------
+
+* [:doc:`/install/centos`][:doc:`/install/almalinux`] Added support for MariaDB 10.9.3.
+
+* [:doc:`/install/ubuntu`] Added support for MariaDB 10.6 on Ubuntu 22.04 (Jammy).
+
```

Comment:
We need to add https://github.com/mroonga/mroonga/commit/95645d8373b6e4763a1cf56f03816d70309fd067.